src/lib/common/avgSampler.cpp

```diff
@@ -285,7 +285,11 @@ double AveragingSampler::getCircularMean() {
   configASSERT(_samples != NULL);
   double sin_sum = getTrigSum(TRIG_MEAN_TYPE_SIN);
   double cos_sum = getTrigSum(TRIG_MEAN_TYPE_COS);
-  return atan2(sin_sum, cos_sum);
+  double mean = atan2(sin_sum, cos_sum);
+  if (mean < 0.0) {
+    mean += 2 * M_PI;
```

Comment:
Ooooh, hey i didn't even try to use M_PI. I think we should use this on line 71 instead of the #define PI value I added:
```
#define PI  3.14159265358979323846
static constexpr double DIRECTION_SAMPLE_MEMBER_MIN = 0.0;
static constexpr double DIRECTION_SAMPLE_MEMBER_MAX = 2*PI;
```

Reply:
It occurred to me to suggest that, but it felt too nitpicky. M_PI is the normal one in the c std lib. :smile:
:+1: if you want to change it.

Reply:
Oh sorry, yeah, I'll change it in this PR.